src/calculateRentalCost.js

```diff
@@ -5,6 +5,15 @@
  */
 function calculateRentalCost(days) {
   // write code here
+  let result = days * 40;
+
+  if (days >= 7) {
+    result = result - 50;
```

Comment:
Remove the `else` keyword as it is not necessary after an `if` block that contains a `return` statement. This simplifies the control flow and improves readability.